app/models/publication.rb

```diff
@@ -157,6 +157,12 @@ def assign_statistics_announcement
     end
   end
 
+  def base_path
+    return "/government/statistics/#{slug}" if statistics? || national_statistic?
```

Comment:
why is this here rather than on the statistics model? I'm sure there's a horrible whitehall reason that statistics and statistics_announcements aren't the same thing but just interested :)

Reply:
There's no model for statistics or national statistics (statistics announcements and statistical data sets are different things).